File: NETMouse - .NET release/Examples/PascalABC.NET/Arrays_14.pas
{$reference ABCNET.dll}
uses ABCNET.Utils;
uses ABCNET.Extensions;
begin
  var a := Arr.ReadInteger(BaseU.ReadInteger('N:'), 'Элемент {0}-ый:');
  var b := a.Where(x -> x mod 2 = 0).ToArray();
  BaseU.ReadChar();
end.

File: NETMouse - .NET release/Examples/C#/Arrays_14.cs
using ABCNET.Utils;
using System.Linq;

namespace TestProgram
{
    internal class Program
    {
        private static void Main(string[] args)
        {
            int[] a = Arr.ReadInteger(Base.ReadInteger("N:"), "Элемент {0}-ый:");
            int[] b = a.Where(x => x % 2 == 0).ToArray();
        }
    }
}